test/fixtures/output/schema-registry/schema/describe-refs-id.golden

```diff
@@ -15,42 +14,44 @@
           "version": 1
         }
       ],
-      "schema": "schema0"
+      "schema": "schema0",
+      "schemaType": "AVRO"
     },
     {
-      "subject": "lvl1-1",
-      "version": 1,
       "id": 1002,
-      "schemaType": "AVRO",
       "references": [
         {
           "name": "ref_lvl2",
           "subject": "lvl2",
           "version": 1
         }
       ],
-      "schema": "schema11"
+      "schema": "schema11",
+      "schemaType": "AVRO",
+      "subject": "lvl1-1",
+      "version": 1
     },
     {
-      "subject": "lvl2",
-      "version": 1,
       "id": 1004,
+      "references": [],
```

Comment:
This looks like a breaking change. `nil` and `[]` are not the same.

Reply:
yes it's introduced in the sdk. refs was a `[]`, now it's a `*[]`

Reply:
Hmm we might need to hardcode a `if [], return nil` here

Reply:
returning `nil` doesn't change the type of refs, it'll still print `[]`. although i think this change would be fine? adding a new line / new column (and empty)

Reply:
Hmm it's not really an additive change in my opinion. That field already exists and customers might have scripts trying to parse it. When they update the CLI, their scripts might not know how to handle `[]` and could panic

Reply:
Using a customized version of schema struct